03.macos/01.overview/docs.en.md

```diff
@@ -29,7 +29,7 @@ AdGuard for Mac is a <i>standalone ad blocker program</i> for macOS computers. W
 
 <img src="https://cdn.adguard.com/public/Adguard/kb/en/Mac-indepth/1-Mainscreen.png" style="max-width: 600px">
     
-The first thing you see upon startup will be a screen with a big On/Off toggle. Various stats of what AdGuard has achieved to date (counting since it’s been installed and switched on) can be found here: the number of trackers, threats and ads blocked, the amount of data saved will be displayed.
+The first thing you see upon startup will be a screen with a big On/Off toggle. Various stats counting since AdGuard has been installed and switched on (or since stats have been reset for the last time) is displayed here: the number of trackers, threats and ads blocked, and the amount of data saved.
```

Comment:
Если "various stats", то "are displayed", наверное. Ну и вообще можно немного переписать предложение: 

"Below are shown various stats (ads blocked, trackers blocked, data saved) since the installation or since the last stats reset".